tests/test_processCcd.py

```diff
@@ -132,20 +132,22 @@ def testProcessCcd(self):
                     print("psf Ixx = %r, Iyy = %r, Ixy = %r" % (psfIxx, psfIyy, psfIxy))
 
                     self.assertEqual(len(icSrc), 28)
-                    self.assertEqual(len(src), 185 + 100)  # 185 real sources plus 100 sky sources
+                    self.assertEqual(len(src), 184 + 100)  # 184 real sources plus 100 sky sources
 
+                    # NOTE: These values are purely empirical, and need to be
+                    # updated to reflect major algorithmic changes.
                     expectedPlaces = 7  # Tolerance for numerical comparisons
                     for name, var, val in [
-                        ("bgMean", bgMean, 191.48635852060525),
-                        ("bgStdDev", bgStdDev, 0.2399466881603354),
-                        ("numGoodPix", numGoodPix, 1966820),
-                        ("imMean", imMean, 1.1237668985230562),
-                        ("imStdDev", imStdDev, 85.81296241298496),
+                        ("bgMean", bgMean, 191.48623786891795),
+                        ("bgStdDev", bgStdDev, 0.23994185672586282),
+                        ("numGoodPix", numGoodPix, 1966606),
```

Comment:
It's frankly beyond this particular PR, but I don't quite follow why ``bgMean``, ``bgStdDev``, ``numGoodPix`` aren't being tested in the `else` condition.